Source/UncompressedVideoSampleProvider.cpp

```diff
@@ -734,22 +734,31 @@ void UncompressedVideoSampleProvider::CheckFrameSize(AVFrame* avFrame)
     }
     else if (hasFormatChanged)
     {
-        // dynamic output size switching
```

Comment:
LOL What do my eyes see? We DID already have dynamic video format switching?? I do not remember a single thing about that. And looking at the history, it was even me who implemented it, some 3 years ago 🤣

Reply:
Well funny story - initially i started off assuming we did not have this code, and was half way through implementing it when I found it and switched the approach. The microsoft approach and your approach seem to have the same effect. Only the audio code is essentially new. I will give it a try to see if we can avoid the whole GUID business and use the setters on the audio stream descriptor. At least it is now more or less confirmed the GUID approach should work. I assumed microsoft did this because the setters in the winRT wrappers were not working correctly, and they were overriding that with the GUID.